src/index.html

```diff
@@ -3,33 +3,28 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello</title>
+    <title>Leaderboard</title>
     <script src="https://kit.fontawesome.com/05c5795d1a.js" crossorigin="anonymous"></script>
 <script defer src="main.js"></script></head>
 <body>
-    <h1>Leaderboard</h1>
-    <div class="big-container">
-        <div class="container-left">
-            <div class="c1r1">
-                <h2>Recent Scores</h2>
-                <button>Refresh</button>
-            </div>
-            <div class="c1r2">
-                <ul id="leaderboard">
-                    <li class="scorer">N1</li>
-                    <li class="scorer">N2</li>
-                    <li class="scorer">N3</li>
-                    <li class="scorer">N4</li>
-                    <li class="scorer">N5</li>
-                </ul>
-            </div>
-        </div>
-        <div class="container-right">
-            <h2>Add your score</h2>
-            <input type="text" id="name" placeholder="Name">
-            <input type="text" id="scorde" placeholder="Score">
-            <button>Add score</button>
-        </div>
-    </div>
+	<h1>Leaderboard</h1>
+	<div class="big-container">
+			<div class="container-left">
+					<div class="c1r1">
+							<h2>Recent Scores</h2>
+							<button id="refreshbtn">Refresh</button>
+					</div>
+					<div class="c1r2">
+							<ul id="leaderboard">
+							</ul>
+					</div>
+			</div>
+			<div class="container-right">
+					<h2>Add your score</h2>
+					<input type="text" id="name" placeholder="Name">
+					<input type="text" id="score" placeholder="Score">
+					<button id="addbtn">Add score</button>
+			</div>
```

Comment:
- [ ] Great job so far 👏🏻 However, According to the project requirements, you should write plain HTML markup with minimum styling. Kindly make sure your HTML code looks like in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), styling is not required for this project


![image](https://user-images.githubusercontent.com/79058364/195183007-4bf728d2-75a0-4782-8c49-e57eaa2048e3.png)

- Make sure to add ramdom player's name and their scrore as shown on the originl template:

![Screenshot from 2022-10-13 04-28-30](https://user-images.githubusercontent.com/79058364/195485720-52107b0d-63ff-4528-a1af-83cbbffa5ae3.png)